helpers/cryptos/lskBaseCoin.js

```diff
@@ -0,0 +1,230 @@
+        recipientAddress: cryptography.getAddressFromBase32Address(address),
+        data,
+        // data: 'Sent with ADAMANT Messenger'
+      },
+      signatures: [],
+    };
+    liskTx.senderPublicKey = this.account.keyPair.publicKey;
+    const minFee = Number(transactions.computeMinFee(this.assetSchema, liskTx)) / this.multiplier;
+
+    return { liskTx, minFee };
+  }
+
+  /**
+   * Builds log message from formed Tx
+   * @param {object} tx Tx
+   * @return {string | object} Log message
+   */
+  formTxMessage(tx) {
+    try {
+      const token = this.token;
+      const status = tx.status ? ' is accepted' : tx.status === false ? ' is FAILED' : '';
+      const amount = tx.amount ? ` for ${tx.amount} ${token}` : '';
+      const height = tx.height ? `${status ? ' and' : ' is'} included at ${tx.height} blockchain height` : '';
+      const confirmations = tx.confirmations ? ` and has ${tx.confirmations} confirmations` : '';
+      const instantSend = !height && !confirmations && tx.instantlock && tx.instantlock_internal ? `${status ? ' and' : ' is'} locked with InstantSend` : '';
+      const time = tx.timestamp ? ` (${utils.formatDate(tx.timestamp).YYYY_MM_DD_hh_mm} — ${tx.timestamp})` : '';
+      const hash = tx.hash;
+      const fee = tx.fee || tx.fee === 0 ? `, ${tx.fee} ${token} fee` : '';
+      const senderId = utils.isStringEqualCI(tx.senderId, this.account.address) ? 'Me' : tx.senderId;
+      const recipientId = utils.isStringEqualCI(tx.recipientId, this.account.address) ? 'Me' : tx.recipientId;
+      return `Tx ${hash}${amount} from ${senderId} to ${recipientId}${status}${instantSend}${height}${time}${confirmations}${fee}`;
+    } catch (e) {
+      log.warn(`Error while building Tx ${tx ? tx.id : undefined} message for ${this.token} of ${utils.getModuleName(module.id)} module: ` + e);
+      return tx;
+    }
+  }
+};
+
+/**
+ * Create axios client to Lisk service
+ * @param {string} url client base url
+ * @return {Object}
+ */
+function createServiceClient(url) {
+  const client = axios.create({ baseURL: url });
+  client.interceptors.response.use(null, (error) => {
+    if (error.response && Number(error.response.status) >= 500) {
+      console.error(`Request to ${url} failed.`, error);
+    }
+    return Promise.reject(error);
+  });
+  return client;
+}
+
+/**
+ * Create axios client to Lisk node
+ * @param {string} url client base url
+ * @return {Object}
+ */
+function createClient(url) {
+  const client = axios.create({ baseURL: url });
+  client.interceptors.response.use(null, (error) => {
+    if (error.response && Number(error.response.status) >= 500) {
+      console.error(`Request to ${url} failed.`, error);
+    }
+    if (error.response && Number(error.response.status) === 404) {
+      if (error.response.data && error.response.data.errors && error.response.data.errors[0] && error.response.data.errors[0].message && error.response.data.errors[0].message.includes('was not found')) {
```

Comment:
You can simplify
`if (error.response?.data?.errors[0]?.message && error.response.data.errors[0].message.includes('was not found'))`